app/services/samples/clone_service.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Samples
+  # Service used to clone samples
+  class CloneService < BaseProjectService
+    CloneError = Class.new(StandardError)
+
+    def execute(new_project_id, sample_ids)
+      validate(new_project_id, sample_ids)
+
+      authorize! @project, to: :clone_sample?
+
+      @new_project = Project.find_by(id: new_project_id)
+      authorize! @new_project, to: :clone_sample_into_project?
+
+      clone_samples(@new_project, sample_ids)
+    rescue Samples::CloneService::CloneError => e
```

Comment:
Call update_metadata_summary after successful cloning

Reply:
Assuming calling the metadata up service is sufficient, like described in the issue. Please see 519150f2a7aa4612c747ca57e339b878225ceb05.